screens/CameraScreen.tsx

```diff
@@ -0,0 +1,181 @@
+import * as React from 'react';
+import { useFocusEffect } from '@react-navigation/native';
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
+import { Camera } from 'expo-camera';
+import { Ionicons, MaterialIcons, MaterialCommunityIcons } from '@expo/vector-icons';
+
+export default function CameraScreen({ navigation: { goBack } }) {
+
+  const [camera, setCamera] = React.useState<Camera | null>(null);
```

Comment:
`useState<Camera>()` will set default `camera` to `undefined`, no need to explicitly use `null`.

Reply:
It gives me this error if I only use <Camera> so it must be <Camera | null> but you're right I don't need to explicitly use null as the initial value.

![image](https://user-images.githubusercontent.com/37185367/111247049-b7a14c00-85dd-11eb-8fbb-5a265e8c64c5.png)

Reply:
I see, then you can leave it as it was, or write `useState<Camera>()` and `setCamera(ref || undefined)`